packages/kbn-shared-ux-components/src/page_template/no_data_page/action_cards/action_cards.test.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+import { shallowWithIntl } from '@kbn/test-jest-helpers';
+import { NoDataCard } from '../no_data_card';
+import { ActionCards } from './action_cards';
+
+describe('ActionCards', () => {
+  const onClick = jest.fn();
+  const action = {
+    recommended: false,
+    button: 'Button text',
+    onClick,
+  };
+  const card = <NoDataCard key={'key'} {...action} />;
```

Comment:
What is the purpose of this children for the test? Can't we have anything rendered? `<div>Card 1</div>` instead of this?